### ISSUE TITLE
False positives for nyc, babel, eslint, webpack and its loaders

### ISSUE THREAD

**Issue by mrchief** [depcheck/depcheck#194]
I've read through all the issues but this cannot get this tool to stop giving out false positives.

For example, I've

`.babelrc`
```
{
  "presets": ["save"],
...
}
```

yet I get `babel-preset-save` as unused. Same goes for several webpack, eslint loaders. 

It also lists out things like nyc, cross-env as unused but they are being used in npm scripts.

I tried `depcheck --specials=webpack,eslint,babel` but no luck. I still get ton of false positives.

What am I missing?

**Comment by lijunle** [comment]
Hi 

Which version depcheck are you using? What is the output with `--json` parameter?

**Comment by mrchief** [comment]
I'm using the latest. 

![image](https://cloud.githubusercontent.com/assets/781818/21288969/b4df4d7c-c45e-11e6-969c-dbff6b07a3e8.png)


```
{
  "dependencies": [
    "babel-eslint",
    "babel-loader",
    "babel-register",
    "babel-runtime",
    "css-loader",
    "file-loader",
    "font-awesome",
    "image-webpack-loader",
    "json-loader",
    "normalize.css",
    "postcss-loader",
    "promise-loader",
    "style-loader"
  ],
  "devDependencies": [
    "eslint-config-airbnb",
    "eslint-loader",
    "eslint-plugin-import",
    "eslint-plugin-jsx-a11y",
    "eslint-plugin-react",
    "postcss-loader",
    "react-addons-test-utils",
    "redbox-react",
    "url-loader",
    "webpack-hot-middleware"
  ],
  "missing": {
    "components": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\app.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\header.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\app.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\header.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\lang-picker.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\profile.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\users.test.js"
    ],
    "styles": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\app.jsx"
    ],
    "utils": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\header.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\users.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\flux\\actions\\locale.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\routes.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\helpers\\mount.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\index.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ],
    "images": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\header.jsx"
    ],
    "flux": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\main.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\helpers\\mount.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\router.jsx"
    ],
    "promise?global!intl": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\intl-loader.js"
    ],
    "promise?global!data": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\intl-loader.js"
    ],
    "data": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\helpers\\mount.js"
    ],
    "pages": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\account.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\server-error.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\not-found.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ],
    "routes": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ]
  },
  "using": {
    "babel-preset-save": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\.babelrc"
    ],
    "babel-plugin-typecheck": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\.babelrc"
    ],
    "babel-plugin-react-transform": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\.babelrc"
    ],
    "react-transform-hmr": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\.babelrc"
    ],
    "react-transform-catch-errors": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\.babelrc"
    ],
    "babel-plugin-transform-react-constant-elements": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\.babelrc"
    ],
    "babel-plugin-transform-react-remove-prop-types": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\.babelrc"
    ],
    "babel-plugin-transform-react-inline-elements": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\.babelrc"
    ],
    "babel-plugin-transform-react-pure-class-to-function": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\.babelrc"
    ],
    "react": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\footer.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\app.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\guides.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\header.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\profile.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\shared\\lang-picker.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\shared\\require-auth.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\shared\\spinner.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\users.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\flux\\stores\\session.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\pages\\account.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\pages\\login.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\pages\\not-found.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\pages\\server-error.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\routes.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\i18n-container.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\i18n-provider.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\intl-loader.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\localized-routes.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\app.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\helpers\\mount.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\router.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\server-html.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ],
    "components": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\app.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\header.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\app.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\header.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\lang-picker.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\profile.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\users.test.js"
    ],
    "styles": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\app.jsx"
    ],
    "connect-alt": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\header.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\profile.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\users.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\pages\\account.jsx"
    ],
    "react-router": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\header.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\users.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\flux\\stores\\session.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\routes.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\localized-routes.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ],
    "utils": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\header.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\users.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\flux\\actions\\locale.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\routes.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\helpers\\mount.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\index.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ],
    "images": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\header.jsx"
    ],
    "lodash": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\profile.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\flux\\createFlux.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\header.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\account.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\profile.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\users.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\dev.config.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\utils\\start-koa.js"
    ],
    "classnames": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\shared\\lang-picker.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\components\\shared\\spinner.jsx"
    ],
    "debug": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\flux\\stores\\locale.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\flux\\stores\\session.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\main.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\i18n-provider.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\image-resolver.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\intl-polyfill.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\dev-server.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\utils\\start-koa.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\utils\\write-stats.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\router.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ],
    "cookies-js": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\flux\\stores\\locale.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\flux\\stores\\session.js"
    ],
    "alt": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\flux\\createFlux.js"
    ],
    "alt-utils": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\flux\\createFlux.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ],
    "babel-polyfill": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\index.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\index.js"
    ],
    "flux": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\main.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\helpers\\mount.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\router.jsx"
    ],
    "react-intl": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\i18n-container.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\i18n-provider.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\intl-loader.js"
    ],
    "promise?global!intl": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\intl-loader.js"
    ],
    "promise?global!data": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\intl-loader.js"
    ],
    "intl": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\app\\utils\\intl-polyfill.js"
    ],
    "babel": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\package.json"
    ],
    "babel-cli": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\package.json"
    ],
    "webpack": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\package.json",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\base.config.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\dev-server.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\dev.config.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\prod.config.js"
    ],
    "ava": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\package.json",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\app.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\header.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\account.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\server-error.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\not-found.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\lang-picker.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\profile.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\users.test.js"
    ],
    "coveralls": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\package.json"
    ],
    "eslint": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\package.json"
    ],
    "nyc": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\package.json"
    ],
    "chai": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\app.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\header.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\account.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\server-error.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\not-found.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\lang-picker.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\profile.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\users.test.js"
    ],
    "chai-enzyme": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\app.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\header.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\account.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\server-error.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\not-found.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\lang-picker.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\profile.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\users.test.js"
    ],
    "enzyme": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\app.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\header.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\helpers\\mount.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\account.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\server-error.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\not-found.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\lang-picker.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\profile.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\users.test.js"
    ],
    "data": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\helpers\\mount.js"
    ],
    "jsdom": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\helpers\\setup-browser-env.js"
    ],
    "pages": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\account.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\server-error.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\pages\\not-found.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ],
    "sinon": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\lang-picker.test.js"
    ],
    "sinon-chai": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\lang-picker.test.js"
    ],
    "faux-jax": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\profile.test.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\test\\users.test.js"
    ],
    "postcss-import": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\base.config.js"
    ],
    "postcss-url": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\base.config.js"
    ],
    "precss": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\base.config.js"
    ],
    "autoprefixer": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\base.config.js"
    ],
    "koa": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\dev-server.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "koa-convert": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\dev-server.js",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "koa-webpack-dev-middleware": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\dev-server.js"
    ],
    "koa-webpack-hot-middleware": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\dev-server.js"
    ],
    "dev-ip": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\dev.config.js"
    ],
    "extract-text-webpack-plugin": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\prod.config.js"
    ],
    "purifycss-webpack-plugin": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\prod.config.js"
    ],
    "browser-sync": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\utils\\start-koa.js"
    ],
    "node-watch": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\internals\\webpack\\utils\\start-koa.js"
    ],
    "babel-core": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\index.js"
    ],
    "react-dom": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\router.jsx",
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ],
    "koa-mount": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "koa-helmet": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "koa-logger": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "koa-favicon": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "koa-static-cache": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "koa-response-time": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "koa-router": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "koa-conditional-get": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "koa-etag": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "koa-compress": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "blocked": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "koa-proxy": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\server\\koa.js"
    ],
    "axios": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\api-client.js"
    ],
    "iso": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ],
    "alt-container": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ],
    "routes": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ],
    "scroll-behavior": [
      "C:\\Git Repo\\isomorphic-flux-boilerplate\\shared\\universal-render.jsx"
    ]
  },
  "invalidFiles": {},
  "invalidDirs": {}
}
```

You can try this repo (my stuff is based on this): https://github.com/iam4x/isomorphic-flux-boilerplate

**Comment by batjko** [comment]
I got the same for all webpack loaders and babel-polyfill, which I am using as an entry point in the webpack config.
Essentially, anything that isn't explicitly imported anywhere.

One caveat: I also briefly got eslint as a false positive, but after another `npm i` that didn't happen anymore.

**Comment by lostpebble** [comment]
Yep, noticing that all my webpack loaders are showing up as unused dependencies, even while using the `specials=webpack` flag.

**Comment by wesleymostien** [comment]
Same here

**Comment by brickfungus** [comment]
In my .babelrc i have these plugins
`"plugins": [
        "react-hot-loader/babel",
        "@babel/plugin-proposal-class-properties",
        "@babel/plugin-proposal-object-rest-spread",
        "@babel/plugin-syntax-dynamic-import",
        "@babel/plugin-transform-modules-commonjs",
        [`

it's good with all the @babel ones, but says react-hot-loader is unused

**Comment by GarrettGeorge** [comment]
The webpack regex looks for webpack files matching `/webpack(\..+)?\.config\.(babel\.)?js/`. If you're files don't match that regex then `--specials=webpack` does nothing.
